encoders/nlp/TransformerTorchEncoder/Dockerfile

```diff
@@ -1,13 +1,30 @@
-FROM jinaai/jina
+FROM continuumio/miniconda3 AS conda
+
+COPY env.yml /
+COPY requirements.txt /
+RUN conda update conda -c conda-forge && \
+    conda env update -f /env.yml -n base && \
+    pip install -r  /requirements.txt --no-cache-dir && \
+    conda clean -afy
+
+FROM ubuntu:20.04
+
+# Prepare shell and file system
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8 SHELL=/bin/bash
+ENV PATH /opt/conda/bin:$PATH
+SHELL ["/bin/bash", "-c"]
+
+# Copy over conda and bashrc, install environment
+COPY --from=conda /opt/ /opt/
+RUN ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
 
 # setup the workspace
 COPY . /workspace
 WORKDIR /workspace
 
-# install the third-party requirements
-RUN pip install -r requirements.txt
-
 # for testing the image
-RUN pip install pytest && pytest && rm -rf '~/.cache/huggingface/transformers/'
+RUN pip install pytest && pytest && rm -rf '/root/.cache/huggingface/transformers/' && rm -rf /tmp/{*,.*}
```

Comment:
can u remove also env.yml and requirements.txt from the root folder (that are copied at the begining? to avoid duplication?

Reply:
They are not part of the image - only the /opt folder is copied from the conda image, everything else is discarded